Tests/test_SeqIO_online.py

```diff
@@ -95,6 +96,19 @@ def funct(d, f, e, l, c):
     del funct
 del database, formats, entry, length, checksum
 
+class UniProtTests(unittest.TestCase):
+    """Test for UniPort module."""
+
+    def test_search(self):
```

Comment:
This generic UniProt search test would be better in a new file I think, say ``test_UniProt_online.py``.

This file (``test_SeqIO_online.py``) was intended for parsing various sequence files from online resources with Bio.SeqIO, as a test of the parser (e.g. has the file format changed in a new release) and of the online wrapper (has the API changed, or is it offline). Could you use the UniProt FASTA return mode here, and any other sequence formats ``SeqIO`` can parse?

Reply:
That looks good, thanks.